pkg/reconciler/taskrun/resources/apply_test.go

```diff
@@ -120,38 +130,29 @@ var (
 				},
 			}},
 		}}},
-	}
-
-	stepTemplateTaskSpec = &v1alpha1.TaskSpec{
-		StepTemplate: &corev1.Container{
-			Env: []corev1.EnvVar{{
-				Name:  "template-var",
-				Value: "$(inputs.params.FOO)",
-			}},
+		Volumes: []corev1.Volume{{
+			Name: "$(inputs.params.FOO)",
+			VolumeSource: corev1.VolumeSource{
+				ConfigMap: &corev1.ConfigMapVolumeSource{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: "$(inputs.params.FOO)",
+					},
+				},
+			}}, {
```

Comment:
nit: indentation might be off here, I'd expect a `}, {` at the same indentation as `Volumes:` above.